### ISSUE TITLE
Ethash hashrate is totally off in v0.18.3

### ISSUE THREAD

**Issue by RainbowMiner** [trexminer/T-Rex#85]
Hi,

```
20201028 23:00:34 [ OK ] 126/127 - 819.01 MH/s, 46ms ... GPU #0
20201028 23:00:40 [ OK ] 127/128 - 818.98 MH/s, 76ms ... GPU #4
20201028 23:00:41 [ OK ] 128/129 - 819.02 MH/s, 46ms ... GPU #1
20201028 23:00:49 [ OK ] 129/130 - 819.11 MH/s, 66ms ... GPU #5
20201028 23:01:18 [ OK ] 130/131 - 818.98 MH/s, 47ms ... GPU #4
```

... the displayed hashrate (and the hashrate in the API) are totally off, in this version.
The above is a testrig with 6 x GTX 1070 which should mine ethash with aprox. 180MH/s . Looking at the pool side, it's about that 180 MH/s what arrives there.

**Comment by trexminer** [comment]
Please, try the latest build 0.18.5.

**Comment by RainbowMiner** [comment]
Seems to be fixed with v0.18.5 - thank you very much!
